### PR TITLE
Search: no results

### PR DESCRIPTION
**Pull request by benisgold**
Fixes TEAM2-475
Figma link (if any): https://www.figma.com/file/SFfYzRoHlapEecS0LTLwhQ/Swap-Aggregator-V2?node-id=646%3A54575

## What changed (plus any additional context for devs)
- new view for when there are no currency search results

## Screen recordings / screenshots

https://user-images.githubusercontent.com/15272675/192884985-e2e64266-d9be-4570-9ac3-eaee4b55c75d.mp4

## What to test
- make sure the text is appropriate depending on the context

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/screens/CurrencySelectModal.tsx [hunk 1]
```diff
@@ -552,6 +552,7 @@ export default function CurrencySelectModal() {
             )}
             {type === null || type === undefined ? null : (
               <CurrencySelectionList
+                onL2={Boolean(currentChainId && currentChainId !== 1)}
```
Comment (estebanmino) [inline src/screens/CurrencySelectModal.tsx:555]: we have `isL2Network` for this

File: src/components/discover-sheet/DiscoverSearch.js [hunk 2]
```diff
@@ -250,6 +250,7 @@ export default function DiscoverSearch() {
       <SearchContainer>
         <CurrencySelectionList
           footerSpacer
+          fromDiscover
```
Comment (estebanmino) [inline src/components/discover-sheet/DiscoverSearch.js:253]: curious to know why this was needed

**Review by estebanmino — COMMENTED** [review]
small comments, looking good on android and ios 🚀 https://www.loom.com/share/6f5d7d440c324208975bc8e553fd7653

File: src/components/discover-sheet/DiscoverSearch.js (hunk 2 above)
Comment (skylarbarrera) [reply src/components/discover-sheet/DiscoverSearch.js:253]: +1,I see what you're trying to do but iirc this prop is for use in swaps and behaves a lil diff than you expect, may want to check currentRoute or pass a different prop here

**Review by skylarbarrera — APPROVED** [review]
smol comment but otherwise lgtm

File: src/components/discover-sheet/DiscoverSearch.js (hunk 2 above)
Comment (skylarbarrera) [reply src/components/discover-sheet/DiscoverSearch.js:253]: after looking a lil deeper, i'm totally fine with this 😆

**Pull request merged**